tmt/steps/provision/mock.py

```diff
@@ -556,10 +567,14 @@ def _do_scrub(self) -> None:
         (self.mock_command_prefix + Command('--scrub=all')).run(cwd=None, logger=self._logger)
 
     def suspend(self) -> None:
-        self.mock_shell.exit_shell()
+        self.stop()
+
+    def start(self) -> None:
+        self.mock_shell.enter_shell()
 
     def stop(self) -> None:
         self.mock_shell.exit_shell()
+        self.run(Command('rm', '-rf', str(self.root_path / 'srv/tmt-mock/*')))
```

Comment:
Could you parametrize/constify `srv/tmt-mock` and add a comment about why the rm command is run here?